lom/src/plugins/vendors/arista/plugin/iptcrc/iptcrc_detection_subs.go

```diff
@@ -0,0 +1,534 @@
+//go:build ignore
```

Comment:
Add comment ; Subscription based implementation is parkd here just because of bug in 4.26 eos related to gnmi subscription feature. Its fixed in 4.29.

Reply:
added